packages/react-scripts/fixtures/kitchensink/src/App.js

```diff
@@ -121,8 +123,12 @@ class App extends React.Component {
   }
 
   render() {
-    const { feature } = this.state
-    if (feature !== null) return <BuiltEmitter feature={feature} />
+    const { feature } = this.state;
+    if (feature !== null) {
+      return (
+        <BuiltEmitter feature={feature} />
+      )
```

Comment:
Still missing a semicolon 😉 
If it's single line, it's fine to just do `return <BuiltEmitter feature={feature} />;`.
I was only referring to cases when we have multiline JSX.